src/app/search/search-workflow-table/search-workflow-table.component.html

```diff
@@ -38,7 +38,9 @@
       >
       <mat-cell fxShow fxHide.lt-sm data-cy="descriptorType" *matCellDef="let workflow" fxFlex="20%">
         <div>
-          <span class="nonclick-badge">{{ workflow?.descriptorType | uppercase }}</span>
+          <span class="nonclick-badge">{{
+            workflow?.descriptorType === 'gxformat2' ? galaxyShortfriendlyName : (workflow?.descriptorType | uppercase)
```

Comment:
I think you could change `'gxformat2'` to `DescriptorLanguageService.knownGalaxyValue`.

Reply:
@coverbeck I think then I will need to make `knownGalaxyValue` `static`, but it will break other code. 
And I feel like it's unnecessary to initialize a private `descriptorLanguageService` and then create `ngOnInit()`, and also make `knownGalaxyValue` `public`?

Reply:
Good point, I hadn't noticed that when I grepped.

It is exposed statically as an enum elsewhere, in ExtendedDescriptorLanguageBean, but you'd still have to do a little extra work on it, so just leave it as is.